sql-database/transparent-data-encryption/setup-tde-byok-sqlmi.ps1

```diff
@@ -35,15 +33,17 @@ Update-AzKeyVaultNetworkRuleSet -VaultName "MyKeyVault" -DefaultAction Deny
 # $key = Add-AzureKeyVaultKey -VaultName MyKeyVault -Name MyTDEKey -Destination Software -Size 2048
```

Comment:
If this is __not__ the recommended way to create a key, it should be moved underneath the uncommented and active script.

Reply:
Moved and changed wording
![image](https://user-images.githubusercontent.com/40007119/55645431-69e2fe80-578d-11e9-8ab7-2b5c2b4cda47.png)